defs.bzl

```diff
@@ -18,20 +18,25 @@ Users should not load files under "/internal"
 """
 
 load("//internal/common:check_bazel_version.bzl", _check_bazel_version = "check_bazel_version")
+load("//internal/common:get_host.bzl", _get_host = "get_host")
 load("//internal/node:node.bzl",
      _nodejs_binary = "nodejs_binary_macro",
      _nodejs_test = "nodejs_test_macro")
 load("//internal/node:node_repositories.bzl",_node_repositories = "node_repositories")
-load("//internal/jasmine_node_test:jasmine_node_test.bzl", _jasmine_node_test = "jasmine_node_test")
+load("//internal/jasmine_node_test:jasmine_node_test.bzl",
+  _jasmine_node_test = "jasmine_node_test",
+  _xjasmine_node_test = "xjasmine_node_test")
 load("//internal/npm_install:npm_install.bzl", _npm_install = "npm_install", _yarn_install = "yarn_install")
 load("//internal/rollup:rollup_bundle.bzl", _rollup_bundle = "rollup_bundle")
 load("//internal/npm_package:npm_package.bzl", _npm_package = "npm_package")
 
 check_bazel_version = _check_bazel_version
+get_host = _get_host
 nodejs_binary = _nodejs_binary
 nodejs_test = _nodejs_test
 node_repositories = _node_repositories
 jasmine_node_test = _jasmine_node_test
+xjasmine_node_test = _xjasmine_node_test
```

Comment:
why expose these in the public API? I don't think this file should be touched in this PR

Reply:
Good point. Its not a useful rule for users 👍